src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2401,20 +2575,37 @@ void AutomationInstrumentClipView::modEncoderButtonAction(uint8_t whichModEncode
 
 				//display only left cursor initially
 				leftPadSelectedX = 0;
-				rightPadSelectedX = kNoLastSelectedPad;
+				rightPadSelectedX = kNoSelection;
+
```

Comment:
code added to re-set mod position when entering pad selection mode to the very left square which corresponds to the pad selection cursor displayed